fundingmanager.go

```diff
@@ -93,6 +93,16 @@ var (
 		"down")
 )
 
+// FundingPeer represents a Lightning Network peer capable of handling funding
+// requests/messages.
+type FundingPeer interface {
+	lnpeer.Peer
+
+	// AddNewChannel adds a new channel to the peer. The channel should fail
+	// to be added if the fundingQuit channel is closed.
```

Comment:
I think `cancel` would be a better name for the chan.

Reply:
Fixed.